src/lib/runDev.js

```diff
@@ -179,37 +183,38 @@ async function runDev (args = [], config, options = {}, log = () => {}) {
       }
       await utils.writeConfig(devConfig.web.injectedConfig, urls)
 
-      log('starting local frontend server ..')
-      const entryFile = path.join(devConfig.web.src, 'index.html')
-
-      // our defaults here can be overridden by the bundleOptions passed in
-      // bundleOptions.https are also passed to bundler.serve
-      const parcelBundleOptions = {
-        cache: false,
-        outDir: devConfig.web.distDev,
-        contentHash: false,
-        watch: true,
-        minify: false,
-        logLevel: 1,
-        ...bundleOptions
-      }
+      if (!options.skipServe) {
+        log('starting local frontend server ..')
+        const entryFile = path.join(devConfig.web.src, '*.html')
```

Comment:
this is the line that causes the 404 @rajarju @purplecabbage I will revert to index.html

Reply:
Of course https://localhost:9080/index.html works but this will break current workflows

Reply:
Parcel issue: https://github.com/parcel-bundler/parcel/issues/1315 
(not sure if I understand it fully)

Reply:
Option added in parcel-bundler@2. we are using parcel-bundler@1
https://github.com/parcel-bundler/parcel/pull/3996

Reply:
Applying PR https://github.com/adobe/aio-cli-plugin-app/pull/324.
The entrypoint fix should go in another PR (not sure what the solution there is, besides updating parcel-bundler)

Reply:
@shazron Thanks for reverting this.